app/src/app/poll/poll.component.ts

```diff
@@ -139,7 +143,7 @@ export class PollComponent implements OnInit, OnDestroy {
     private session: SessionService,
     private formBuilder: FormBuilder,
     private title: AppTitleService,
-    private notif: PollNotifService,
+    private notif: PollNotifService
```

Comment:
In this project, we want to have a comma at the end of each line of a list, even the last one. There are two reasons for that:
 1. It makes future modifications easier, if someone wants to add or remove items, or if she wants to reoder the items.
 2. In Go (the language used in the server part of the application), these commas are mandatory, and it is better to have the same conventions for all parts of the application.